poller-lambdas/src/aws.ts

```diff
@@ -1,28 +1,8 @@
 import { SecretsManagerClient } from '@aws-sdk/client-secrets-manager';
-import { SendMessageCommand, SQSClient } from '@aws-sdk/client-sqs';
+import { SendMessageCommand } from '@aws-sdk/client-sqs';
 import { POLLER_LAMBDA_ENV_VAR_KEYS } from '../../shared/pollers';
-import {
-	getEnvironmentVariableOrCrash,
-	isRunningLocally,
-	remoteAwsConfig,
-} from './config';
-
-const buildLocalFakeSqsClient = () => {
-	const queueData: Record<string, SendMessageCommand[]> = {};
-	return {
-		queueData: queueData,
-		send: (command: SendMessageCommand) => {
-			const currentQueue: SendMessageCommand[] =
-				queueData[command.input.QueueUrl!] ?? [];
-			queueData[command.input.QueueUrl!] = [...currentQueue, command];
-			return Promise.resolve();
-		},
-	};
-};
-
-export const sqs = isRunningLocally
```

Comment:
`sqs` client construction is being moved to `../../shared` because the queueing lambda needs to use it too now